workspaces/vscode-python/Dockerfile

```diff
@@ -1,7 +1,8 @@
+ARG BASE_IMAGE_REGISTRY=docker.io
```

Comment:
Same as other comment, I think this will be cleaner using build contexts.

Reply:
Done in 582c00c.